test/remixing.ts

```diff
@@ -0,0 +1,49 @@
+import { ethers } from 'hardhat'
```

Comment:
ignore this file for now just a script that helps me test directly without dealing with fixtures